trade-finance-manager-api/api-tests/v1/deal-cancellation/deal-cancellation.get.api-test.ts

```diff
@@ -0,0 +1,121 @@
+import { AnyObject, TEAM_IDS, TfmDealCancellation } from '@ukef/dtfs2-common';
+import { ObjectId } from 'mongodb';
+import { createApi } from '../../api';
+import app from '../../../src/createApp';
+import { initialiseTestUsers } from '../../api-test-users';
+import { TestUser } from '../../types/test-user.ts';
+import { withTeamAuthorisationTests } from '../../common-tests/with-team-authorisation.api-tests.ts';
+
+const getDealCancellationMock = jest.fn() as jest.Mock<Promise<TfmDealCancellation>>;
+
+jest.mock('../../../src/v1/api', () => ({
+  ...jest.requireActual<AnyObject>('../../../src/v1/api'),
+  getDealCancellation: () => getDealCancellationMock(),
+}));
+
+const originalProcessEnv = { ...process.env };
+const { as, get } = createApi(app);
+
+const validId = new ObjectId().toString();
+
+describe('/v1/deals/:id/cancellation', () => {
+  let testUsers: Awaited<ReturnType<typeof initialiseTestUsers>>;
+  let aPimUser: TestUser;
+
+  describe('GET /v1/deals/:id/cancellation', () => {
+    beforeEach(() => {
+      jest.resetAllMocks();
+    });
+
+    beforeAll(async () => {
+      testUsers = await initialiseTestUsers(app);
+      aPimUser = testUsers().withTeam(TEAM_IDS.PIM).one();
+    });
+
+    afterAll(() => {
+      jest.restoreAllMocks();
+      process.env = originalProcessEnv;
+    });
+
+    const getTfmDealCancellationUrl = ({ id }: { id: string }) => `/v1/deals/${id}/cancellation`;
```

Comment:
Could we create a DRY test helper for this, noticed it's repeated and with different names:

```ts
const getTfmDealCancellationUrl = ({ id }: { id: string }) => `/v1/deals/${id}/cancellation`;
const getTfmDealCancellationUpdateUrl = ({ id }: { id: string }) => `/v1/deals/${id}/cancellation`;
```